projects/step-frontend/src/lib/modules/timeseries/ranger/ts-ranger.component.ts

```diff
@@ -84,10 +84,12 @@ export class TSRangerComponent implements OnInit, AfterViewInit, OnChanges {
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    const settings = changes['settings'];
+    let settings = changes['settings'];
```

Comment:
Why consts have been changed to `let` here.
If value is immutable inside the code block (method boy, if statement, cycle body, etc) - it should be const.
If you need to reassign it - make it `let`.